scss/atoms/typography/_mixins.scss

```diff
@@ -0,0 +1,26 @@
+// ==========================================================================
```

Comment:
@MichaelArestad Your right sorry i looked this up and misread. You are correct here.

So in this case there is basically one major case that makes sense to use single line comments which is where we have a comment that would end up with no context and make no sense if it ended up in an uncompressed output file. This would likely be the case for most functions. Most other places should probably stick to multiline i think still. Since we have both css and scss files in this repo i would like to try to keep things as clear and consistent as possible.

@MichaelArestad how does that sound to you?